config/form_profile_mappings/10-7959F-1.yml

```diff
@@ -0,0 +1,22 @@
+
+veteran:
+  date_of_birth: [identity_information, date_of_birth]
+  full_name: [identity_information, full_name]
+  first: [identity_information, first]
+  middle: [identity_information, middle]
+  last: [identity_information, last]
+physical_address:
+  country: [contact_information, country]
+  street: [contact_information, street]
+  city: [contact_information, city]
+  state: [contact_information, state]
+  postal_code: [contact_information, postal_code]
+mailing_address:
+  country: [contact_information, country]
+  street: [contact_information, street]
+  city: [contact_information, city]
+  state: [contact_information, state]
+  postal_code: [contact_information, postal_code]
```

Comment:
@michaelclement This might have to be adjusted in the future as right now I have physical and mailing the same but I want to see how it works first before I start making customizations if you're okay with that.

Reply:
Yeah that seems reasonable to me 👍

Reply:
@balexandr Can you confirm this works? I would suggest reverting contact_information.street and etc, does not exist. contact_information.address.street does.